synapse/lib/storm.py

```diff
@@ -1734,3 +1735,18 @@ def _stormOperDelProp(self, query, oper):
 
         nodes = query.take()
         [query.add(core.delTufoProp(n, prop)) for n in nodes]
+
+    def _stormOperGetOpers(self, query, oper):
+        '''
+        Get a list of the storm operators available to the current cortex.
+
+        Args:
+            query (s_storm.Query): Query object
+            oper ((str, dict)): Operator tufo
+
+        Returns:
+            None
+        '''
+        for oper, func in self.operfuncs.items():
+            node = s_tufo.ephem('oper', oper, func=str(func))
+            query.add(node)
```

Comment:
It would be prettier if used `func='%s.%s' % (func.__module__, func.__name__)` (otherwise function address will appear which is kinda gross)